cisco_secure_endpoint/assets/logs/cisco-secure-endpoint_tests.yaml

```diff
@@ -0,0 +1,262 @@
+id: cisco-secure-endpoint
+tests:
```

Comment:
For all the samples I do not see any example from the 2 sub pipelines you have defined (`service:event` and `service:audit`).
You need to create the sample and specify the service next to the sample [like done here](https://github.com/DataDog/integrations-core/blob/master/cisco_duo/assets/logs/cisco-duo_tests.yaml#L4-L47).

I am not sure all this sample are not supposed to be reprocessed at all otherwise? Are they some real case of logs sent for this technology?

Reply:
Thankyou for pointing this out. I have added the service and also added one more sample log to cover event service logs. 
These samples should have been processed, and it seems that they were not parsed correctly before. 
I have made the required changes, and now they are processed correctly.

Reply:
Great, thank you!